developer/src/kmc-ldml/src/util/pattern-parser.ts

```diff
@@ -0,0 +1,116 @@
+/**
```

Comment:
This file is here and in common/web/types as well?

Reply:
yes, that's corrected in a downstream PR

Reply:
ok... for future reviews, please add a PR comment for things like this so it saves effort and mental load for reviewers 😁